src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+* Excluded attributes corresponding to `LogRecord.EventId`,
+`LogRecord.CategoryName` and `LogRecord.Exception` from the exported data. This
+is done in order to move ahead with stable release of exporter. The attributes
```

Comment:
> This is done in order to move ahead with stable release of exporter.

I don't think we need to have this on the CHANGELOG. It's better to cite the reason as the convention for exporting these attributes not being stable.